dms-application/src/main/kotlin/team/aliens/dms/common/dto/PageData.kt

```diff
@@ -1,13 +1,17 @@
 package team.aliens.dms.common.dto
 
-data class PageData(
-    val page: Long = 0,
-    val size: Long = 10000
+class PageData(
+    page: Long?,
+    size: Long?
 ) {
+
+    val page: Long = page ?: 0
+    val size: Long = size ?: 500
```

Comment:
따로 구분한 이유가 있나요?

Reply:
null로 들어왔을때 처리하기 위함입니다

Reply:
그냥 아래처럼 하면 되는 것 아닌가요?
```suggestion
    val page: Long? = 0
    val size: Long? = 500
```

Reply:
아래 테스트 결과가 성공합니다. null을 생성자에 대입했을땐 null로 들어갑니다. 
![image](https://user-images.githubusercontent.com/81006587/219959415-bb68a583-f83e-40cf-b2f8-5bd67ff11a95.png)

Reply:
흠 근데 생각해보니 그냥 nullable 타입 빼고 기본 값 적용시키면 되는 것 아닌가요??

Reply:
ModelAttrbute 쓰지 않고 RequestPram으로 그냥 page랑 size 각각 받아서 기본값 지정하는게 나을까요?

Reply:
하나로 묶는 게 나을 것 같아서 그럼 기존 방식을 유지하는 걸로 할까요?

Reply:
넵 알겠습니다!